keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactsSummary.kt

```diff
@@ -10,12 +11,15 @@ data class EnvironmentArtifactsSummary(
 data class ArtifactVersions(
   val name: String,
   val type: ArtifactType,
+  val statuses: Collection<ArtifactStatus>,
```

Comment:
Not a `Set`? Also in `DebianArtifact` I see it's a `List`, which seems like it should also be a `Set` as there's no point in allowing redundant status on the collection.

Reply:
Good call